lisa/util/process.py

```diff
@@ -180,17 +189,23 @@ def _process_command(
                 split_command = ["cmd", "/c", command]
             else:
                 split_command = []
+                if nohup:
```

Comment:
The order is different with the else. `sudo` and `nohup`, which one should be the first one?

Reply:
I did this too, these look reversed but they're not. The first is 'push front' since we're appending to an empty list, the other is push back since we're prefixing the string with a new command.

Reply:
nohup should be first though (except in cases where nohup is not present as a binary)